### PR TITLE
split coordinator servicehost into poolhost and workhost

### PR DESCRIPTION
**Pull request by kidsbear**
This for #200 
Any suggestions for how to test it? I am not sure how to add coordinator arguments and how to check the output. (I can't find any exiting tests for coordinator parameters)

### CODE REVIEW

File: projects/coordinator/src/main/java/org/batfish/coordinator/config/Settings.java [hunk 1]
```diff
@@ -34,9 +34,10 @@
    private static final String ARG_QUEUE_COMPLETED_WORK = "qcompletedwork";
    private static final String ARG_QUEUE_INCOMPLETE_WORK = "qincompletework";
    private static final String ARG_QUEUE_TYPE = "qtype";
-   private static final String ARG_SERVICE_HOST = "servicehost";
 
+   private static final String ARG_SERVICE_POOL_HOST = "poolhost";
```
Comment (dhalperi) [inline projects/coordinator/src/main/java/org/batfish/coordinator/config/Settings.java]: from #200, shouldn't these be called "pullbindhost" and "workbindhost"?

Comment (dhalperi) [inline projects/coordinator/src/main/java/org/batfish/coordinator/config/Settings.java]: and same in the local variable names below.

Comment (kidsbear) [inline projects/coordinator/src/main/java/org/batfish/coordinator/config/Settings.java]: I was trying to keep names consistent, the surrounding names are 'servicehost', 'poolport', 'workport', I can change them to 'poolbindhost'.

File: projects/coordinator/pom.xml [hunk 2]
```diff
@@ -141,5 +141,15 @@
       <artifactId>slf4j-jdk14</artifactId>
       <scope>runtime</scope>
     </dependency>
+      <dependency>
```
Comment (dhalperi) [inline projects/coordinator/pom.xml]: looks like the indent is messed up here (did you use tabs/spaces by any chance)

please also copy the comment from other pom.xml that explains that below this line is test dependencies.

File: projects/coordinator/src/main/java/org/batfish/coordinator/Main.java [hunk 3]
```diff
@@ -80,7 +80,7 @@ private static void initPoolManager() {
 
       if (_settings.getSslPoolDisable()) {
          URI poolMgrUri = UriBuilder
-               .fromUri("http://" + _settings.getServiceHost())
+               .fromUri("http://" + _settings.getServicePoolHost())
```
Comment (dhalperi) [inline projects/coordinator/src/main/java/org/batfish/coordinator/Main.java]: this should be `getPoolBindHost`

File: projects/coordinator/src/main/java/org/batfish/coordinator/Main.java [hunk 4]
```diff
@@ -115,7 +115,7 @@ private static void initWorkManager() {
 
       if (_settings.getSslWorkDisable()) {
          URI workMgrUri = UriBuilder
-               .fromUri("http://" + _settings.getServiceHost())
+               .fromUri("http://" + _settings.getServiceWorkHost())
```
Comment (dhalperi) [inline projects/coordinator/src/main/java/org/batfish/coordinator/Main.java]: and `getWorkBindHost`.

File: projects/coordinator/src/main/java/org/batfish/coordinator/config/Settings.java [hunk 5]
```diff
@@ -80,8 +81,9 @@
    private String _queueCompletedWork;
    private WorkQueue.Type _queueType;
    private String _queuIncompleteWork;
-   private String _serviceHost;
+   private String _servicePoolHost;
```
Comment (dhalperi) [inline projects/coordinator/src/main/java/org/batfish/coordinator/config/Settings.java]: `_poolBindHost`.

File: projects/coordinator/src/main/java/org/batfish/coordinator/config/Settings.java [hunk 6]
```diff
@@ -80,8 +81,9 @@
...
    private int _servicePoolPort;
+   private String _serviceWorkHost;
```
Comment (dhalperi) [inline projects/coordinator/src/main/java/org/batfish/coordinator/config/Settings.java]: `_workBindHost`.

File: projects/coordinator/src/main/java/org/batfish/coordinator/config/Settings.java [hunk 7]
```diff
@@ -274,8 +280,9 @@ private void initConfigDefaults() {
       setDefaultProperty(ARG_QUEUE_COMPLETED_WORK, "batfishcompletedwork");
       setDefaultProperty(ARG_QUEUE_INCOMPLETE_WORK, "batfishincompletework");
       setDefaultProperty(ARG_QUEUE_TYPE, WorkQueue.Type.memory.toString());
-      setDefaultProperty(ARG_SERVICE_HOST, "0.0.0.0");
+      setDefaultProperty(ARG_SERVICE_POOL_HOST, "0.0.0.0");
       setDefaultProperty(ARG_SERVICE_POOL_PORT, CoordConsts.SVC_CFG_POOL_PORT);
+      setDefaultProperty(ARG_SERVICE_WORK_HOST, "0.0.0.0");
```
Comment (dhalperi) [inline projects/coordinator/src/main/java/org/batfish/coordinator/config/Settings.java]: same here for the constant names...

File: projects/coordinator/src/test/java/org/batfish/coordinator/config/SettingsTest.java [hunk 8]
```diff
@@ -0,0 +1,50 @@
+package org.batfish.coordinator.config;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.Test;
+
+/**
+ * Tests for {@link Settings}
```
Comment (dhalperi) [inline projects/coordinator/src/test/java/org/batfish/coordinator/config/SettingsTest.java]: Javadoc should end in a period.

File: projects/coordinator/src/test/java/org/batfish/coordinator/config/SettingsTest.java [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+   }
+
+   @Test
+   public void testGetServiceWorkHost() throws Exception {
+      String[] args = new String[]{
+            "-workbindhost=10.10.10.10"
+      };
+      Settings settings = new Settings(args);
+      assertThat(settings.getServiceWorkHost(), equalTo("10.10.10.10"));
+   }
```
Comment (dhalperi) [inline projects/coordinator/src/test/java/org/batfish/coordinator/config/SettingsTest.java]: drop these two middle tests :)

**Pull request merged**